clients/feeder/transaction.go

```diff
@@ -15,9 +15,9 @@ const (
 
 func (es *ExecutionStatus) UnmarshalJSON(data []byte) error {
 	switch string(data) {
-	case "SUCCEEDED":
+	case "\"SUCCEEDED\"":
```

Comment:
maybe use backticks? i.e. 
```
case `"SUCCEEDED"`
```
but it's up to you

Reply:
Yeap, much better. Thanks for pointing that out.